src/core/server/base.js

```diff
@@ -348,7 +349,10 @@ function baseServer(
         const token = req.universalCookies.get(config.get('cookieName'));
         if (token) {
           store.dispatch(setAuthToken(token));
+        } else {
+          store.dispatch(fetchSiteStatus());
```

Comment:
If someone were grepping the code for `fetchSiteStatus` they might arrive at this and think it's a bug that this is *only* called without a token (tokens have no obvious connection to site status). I suggest adding a comment saying we only need to do this without a token because the user login saga already sets the site status.